hyperspy/drawing/_markers/point.py

```diff
@@ -87,7 +87,7 @@ def update(self):
                                  self.get_data_position('y1')])
         self.marker._sizes = [self.get_data_position('size')]
 
-    def plot(self):
+    def plot(self, fig_draw_animated=True):
```

Comment:
To be honest, I not 100% sure what `ax.hspy_fig._draw_animated()` does. I'm guessing it tries to draw every animated object which hasn't already been drawn?